### PR TITLE
Issue-141: Turn on jwt validation for host-api

### PR DESCRIPTION
**Pull request by pranavs18**
Added the new parameters for turning on jwt validation for host-api

### CODE REVIEW

**Comment by pranavs18** [comment]
PR for py-agent code for enabling JWT-Auth

**Comment by ibuildthecloud** [comment]
Please rebase

File: cattle/plugins/host-api/host_api.py [hunk 1]
```diff
@@ -16,8 +18,12 @@ def on_startup(self):
                     '-cadvisor-url',  url,
                     '-logtostderr=true',
                     '-ip', Config.host_api_ip(),
-                    '-port', str(Config.host_api_port())],
+                    '-port', str(Config.host_api_port()),
+                    '-auth=true',
+                    '-host-uuid', DockerConfig.docker_uuid(),
+                    '-public-key', Config.jwt_public_key_file()],
                    env=env)
+        logging.info("Turning on JWT-Authentication")
```
Comment (ibuildthecloud) [inline cattle/plugins/host-api/host_api.py]: No need for this log statement, besides that fact the message is wrong.  What is being done here is that the Host API is being started.

Comment (pranavs18) [inline cattle/plugins/host-api/host_api.py]: Well, my idea to put that log statement was that we are now passing a JWT token (api.crt) from here when the HOST-API is being launched.

Comment (pranavs18) [inline cattle/plugins/host-api/host_api.py]: Anyways, i'll remove that.

Comment (pranavs18) [inline cattle/plugins/host-api/host_api.py]: Updated the PR as per the comments.

**Comment by pranavs18** [comment]
Rebasing done.

**Pull request merged**